ffc-pay-event/index.js

```diff
@@ -0,0 +1,18 @@
+const { saveEvent } = require('./event')
+const { sendMessage } = require('./signalr')
+const { validateEvent } = require('./event-schema')
+
+module.exports = async function (context, message) {
+  const event = message
+  context.log.info(`Received event: ${JSON.stringify(event)}`)
+  try {
```

Comment:
do we need this try catch if all we're doing is logging the error?  The function by default will log the error for us if one is thrown.